src/main/charts/bitbucket/templates/NOTES.txt

```diff
@@ -52,4 +52,4 @@ Get the {{ title .Chart.Name }} URL by running these commands in the same shell:
 #################################################################################
 {{- end }}
 
-For further documentation, see https://github.com/atlassian-labs/data-center-helm-charts/tree/master/docs
\ No newline at end of file
+For further documentation, see https://github.com/atlassian-labs/data-center-helm-charts/tree/main/docs
```

Comment:
Please revert the changes from all `NOTES.txt` files, I will update the files to point to our github pages instead.

Reply:
This is the PR:
https://github.com/atlassian-labs/data-center-helm-charts/pull/233